src/vulkan-renderer/application.cpp

```diff
@@ -16,57 +16,44 @@ static void frame_buffer_resize_callback(GLFWwindow *window, int width, int heig
     app->frame_buffer_resized = true;
 }
 
-VkResult Application::load_TOML_configuration_file(const std::string &TOML_file_name) {
-    spdlog::debug("Loading TOML configuration file: '{}'.", TOML_file_name);
+VkResult Application::load_toml_configuration(const std::string &file_name) {
+    spdlog::debug("Loading TOML configuration file: '{}'", file_name);
 
-    std::ifstream toml_file;
-
-    // Check if this file exists.
-    toml_file.open(TOML_file_name.c_str(), std::ios::in);
-
-    if (!toml_file.is_open()) {
-        spdlog::error("Could not open configuration file: '{}'!", TOML_file_name);
+    std::ifstream file(file_name, std::ios::in);
+    if (!file) {
+        spdlog::error("Could not open configuration file: '{}'!", file_name);
         return VK_ERROR_INITIALIZATION_FAILED;
     }
 
-    toml_file.close();
-
     // Load the TOML file using toml11.
-    auto renderer_configuration = toml::parse(TOML_file_name);
+    auto renderer_configuration = toml::parse(file);
```

Comment:
This is interesting. In theory this should work fine on every system shouldn't it?